### ISSUE TITLE
Component Notify undeclared

### ISSUE THREAD

**Issue by AubSs** [element-plus/element-plus#523]
Notify is not define in `this`

V: "element-plus": "^1.0.1-alpha.11",

--- error (typescript)
```
Failed to compile.

TS2339: Property '$notify' does not exist on type 'ComponentPublicInstance<{}, {}, {}, {}, {}, EmitsOptions, {}, {}, false, ComponentOptionsBase<{}, {}, {}, {}, {}, ComponentOptionsMixin, ComponentOptionsMixin, EmitsOptions, string, {}>>'.
    10 |     },
    11 |     created() {
  > 12 |         this.$notify.success({
       |              ^^^^^^^
    13 |           title: 'Success',
    14 |           message: 'This is a success message',
    15 |           offset: 100
```

--- Code sample
```
<template>
    <div>test</div>
</template>

<script lang="ts">
import { defineComponent } from 'vue';

export default defineComponent({
    components: {
    },
    created() {
        this.$notify.success({
          title: 'Success',
          message: 'This is a success message',
          offset: 100
        });
    }
});
</script>
```

**Comment by Ryan2128** [comment]
In my test, it work well. Did you import element-plus correctly?

**Comment by JeremyWuuuuu** [comment]
You need to provide us a minimal reproducible version unless we have no clue what caused this issue.
Firstly you need to check if you registered `ElementPlus` correctly, then we can have further discussion(provide a minimal reproducible code)

**Comment by AubSs** [comment]
[test.zip](https://github.com/element-plus/element-plus/files/5475039/test.zip)

It's really a simple install

V: alpha 11

**Comment by JeremyWuuuuu** [comment]
For alpha.12 we have 'element-plus/lib/*' exported. You may import `Nofity` directly from `element-plus/lib/notification' package.

**Comment by zazzaz** [comment]
Thanks, we will fix this bug in the next few days in #630

**Comment by zazzaz** [comment]
fixed in 1.0.1-alpha.18
